src/main/java/org/prebid/server/handler/VtrackHandler.java

```diff
@@ -189,6 +190,7 @@ private static void respondWithServerError(RoutingContext routingContext, String
     }
 
     private static void respondWith(RoutingContext routingContext, HttpResponseStatus status, String body) {
+        routingContext.response().putHeader(HttpUtil.CONTENT_TYPE_HEADER, HttpHeaderValues.APPLICATION_JSON);
```

Comment:
This can be done in lambda below.

Reply:
idk why, but having this done in lambda crashes test